<launch>
	<arg name="image_sub_topic_name" default="/kinect/qhd/image_color_rect" />
	<arg name="debug" default="false" />

	<node pkg="rail_object_detector" type="object_detector_node.py" name="object_detector_node">
		<param name="image_sub_topic_name" type="string" value="$(arg image_sub_topic_name)" />
		<param name="debug" type="bool" value="$(arg debug)" />
	</node>
</launch>